src/map/Map.ts

```diff
@@ -0,0 +1,123 @@
+import Phaser from "phaser";
+
+const ROOT_MAP_FOLDER = 'map';
+
+const TILE_WIDTH = 32;
+const TILE_HEIGHT = 32;
+
+const TILE_MARGIN = 0;
+const TILE_SPACING = 0;
+
+const MAP_KEY = ROOT_MAP_FOLDER;
+const TILE_SHEET_KEY = 'tileSheet';
+const TILE_SHEET_NAME = 'tiles'; // name of the tiles in the Tiled programme
+
+const MAP_DATA_FILE = 'mapData.json';
+const TILE_SHEET_FILE_PATH = 'tileset.png';
```

Comment:
is it currently possible to change these dynamically or are they hardcoded?

might be good if this Map loader class could load different maps for different scenes